DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/AnonymizerConfigurationManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.IO;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.AnonymizerConfigurations;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core
+{
+    public sealed class AnonymizerConfigurationManager
```

Comment:
>AnonymizerConfigurationManager [](http://example.com/codeflow?start=24&length=30)

just curious, all the functions in this file may throw an exception, should we use AnonymizationConfigurationException instad of throwing the original ones?

Reply:
throw AnonymizerConfigurationException instead of original JsonException.